src/nitro.ts

```diff
@@ -141,6 +142,11 @@ export async function setupNitroBridge () {
   // Let nitro handle #build for windows path normalization
   delete nitroConfig.alias['#build']
 
+  if (nuxt.options.generate.routes instanceof Function) {
+    console.warn('It is recommended to migrate the `generate.routes` function to the `nitro:config` hook instead. See https://github.com/nuxt/bridge/pull/475.')
```

Comment:
We might intoroduce better nitro hook (in prerender: namespace) to allow extending async hooks. nitro:config is also for dev and nornal build.

Reply:
Agreed - I think it would be good to have such a hook. Updating this section to avoid calling it in dev/prepare.